app/assets/stylesheets/components/common/_footer_secondary.scss

```diff
@@ -103,8 +104,33 @@
   }
 }
 
+.footer-secondary__sfgb-text {
+  display: inline-block;
+  & > * {
+    display: block;
+  }
+}
+
+.footer-secondary__sfgb-link {
+  margin-top: $baseline-unit*2;
+}
+
 .accessibility-feedback-link .icon--accessibility {
   position: relative;
   top: 2px;
   margin-right: 4px;
 }
+
+.footer-secondary--inner {
+  @include column(12);
+  @include respond-to($mq-l) {
+    display: -ms-flex;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
Probably going to need the prefixes for justify-content and align-items in ie
-ms-flex-align: center;
-ms-flex-pack: justify;

Reply:
@jiveturkeyJay - added these in, are you happy with the way I've ordered all these styles?

Reply:
Looks good to me